cumulusci/tasks/marketing_cloud/deploy.py

```diff
@@ -73,12 +72,23 @@ def _run_task(self):
             "Authorization": f"Bearer {self.mc_config.access_token}",
             "SFMC-TSSD": self.mc_config.tssd,
         }
+        custom_endpoint = self.options.get("endpoint")
+        self.endpoint = (
+            custom_endpoint
+            if custom_endpoint
+            else MCPM_ENDPOINT.format(
+                get_mc_stack_key(self.mc_config.tssd, self.mc_config.access_token)
+            )
+        )
```

Comment:
I know this is a little gross. I'm trying to cheat so I don't need an if/else block which would require another test.